### ISSUE TITLE
Offer APK download

### ISSUE THREAD

**Issue by IzzySoft** [dsolonenko/financisto#56]
Would you mind attaching the `.apk` file to the `releases/`, so folks without Playstore can grab it? I'd then offer to make it available via [my F-Droid compatible repository](https://apt.izzysoft.de/fdroid/repo) (including providing updates as soon as they are found here).

Of course, bringing Financisto to the official F-Droid repo would be much better even :wink:

**Comment by n1trux** [comment]
yes, please, @dsolonenko, support it. I'd be willing to pay a bug bounty for this.

**Comment by paulle** [comment]
Please, I would like it too seeing this nice app on f-droid.

**Comment by IzzySoft** [comment]
Any chance?

**Comment by dsolonenko** [comment]
Here's a bit of cumbersome way of finding the latest apk to download: Go to the latest Bitrise build for the master, say, https://app.bitrise.io/build/8cd442cdfcca4ce6. Open the logs for `deploy-to-bitrise-io` step and look for `The public install page url is now available in the Environment Variable: BITRISE_PUBLIC_INSTALL_PAGE_URL (value: https://app.bitrise.io/artifact/27095012/p/b349caf4591cac3b6ab50e982bcd00dd)`. Open the URL and you'll see the APK download there.

**Comment by IzzySoft** [comment]
Thanks, Denis – but for the "average Joe", that's impractical: apart from the fact the approach is easily forgotten, Joe is already lost in the first step (identifying the build).

What speaks against having your app listed at F-Droid? Ah well, I see: you're [using GMS](https://github.com/dsolonenko/financisto/blob/master/app/build.gradle#L87), which is proprietary and thus not accepted at F-Droid. But if it's non-essential (G+ is dead, and Drive is just a cloud storage while you also have Dropbox there for that), maybe there could be a build flavor without that. Or, if that's too much trouble, if you'd just continue tagging (1.8.1 surely isn't the latest release if the `build.gradle` says 1.8.3) and could attach the APK to its corresponding tag?
